ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3589,6 +3589,58 @@ router_l2_vpn:
     prefix_list: < prefix_list_name >
 ```
 
+### Router MSDP
+
+```yaml
+router_msdp:
+  originator_id_local_interface: < interface >
+  rejected_limit: < 0-40000 >
+  forward_register_packets: < true | false >
+  connection_retry_interval: < 1-65535 >
+  group_limits:
+    - limit: < 0-40000 >
+      source_prefix: < ipv4_prefix/mask >
```

Comment:
I expect this list of dicts can not have the same source_prefix multiple times? Please move it on top and set it as `primary_key` in the schema. Thanks.

Reply:
Fixed!